src/ode_solve.jl

```diff
@@ -330,10 +331,24 @@ function generate_loss(strategy::WeightedIntervalTraining, phi, f, autodiff::Boo
     return loss
 end
 
+function generate_additional_points_loss(tstops, phi, f, autodiff::Bool, tspan, p, batch)
```

Comment:
```suggestion
function generate_additional_points_loss(phi, f, autodiff::Bool, tstops, p, batch)
```

Follow the normal ordering you see elsewhere. No reason to just mix things up.

Reply:
Fixed